addons/html_builder/static/src/plugins/card_option.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.CardOption">
+    <BuilderRow label.translate="Card Width">
+        <BuilderRange
+            id="'card_width'"
+            action="'setCardWidth'"
+            min="8"
+            max="100"
+            displayRangeValue="true"
+            computedOutput="(value) => value + '%'" />
```

Comment:
unit="%"
(You maybe not need computedOutput need to check)

Reply:
apparently, still needed...